### DISCUSSION TITLE
trouble understanding coordinate values

### DISCUSSION THREAD

**Discussion by esovetkin** [cityjson/specs#163] [Q&A] [answered]
Hi, I have trouble understanding coordinate system values. My original data comes in `citygml` format, which I transform to `cityjson` file using `citygml-tools`, and then to an `obj` file using `cjio` library. 

However, during one of the steps my second coordinate value changes its range.

Here is an example of citygml file:
```
> cat ex.gml
...
 <gml:posList srsDimension="3">280962.724 5657686.942 38.017 ...
...
```
that has coordinates in `epsg:25832`.

citygml-tools generates cityjson file, that has seemingly correct coordinate system:
```
> cat ex.json
...
"geographicalExtent": [
       280000,
       5657000,
       0,
       281000,
       5658000,
       50.072
     ],
     "referenceSystem": "https://www.opengis.net/def/crs/EPSG/0/25832"
...
```

However, if I export cityjson to an obj file, or use cjio to work with geometries, the second coordinate changes its range from "5658*" to "9167*"
```
cat ex.obj
 v 280962.724 9167323.573 42.436
 v 280971.022 9167319.498 42.436
 v 280973.051 9167323.645 47.246
...
```

I have trouble understanding why this happens, and I would like to keep by coordinate system intact.

**Comment by esovetkin** [comment, marked as answer]
Ah, I understood that the way coordinates are stored does not equal the actual values of the coordinates (despite visual similarities).

https://cjio.readthedocs.io/en/latest/api_tutorial_basics.html#Geometry-boundaries-and-Semantic-Surfaces

This explains how to get the original coordinates.

**Reply by kenohori** [reply]
Also see: https://www.cityjson.org/dev/realcoords/
